modules/default/updatenotification/node_helper.js

```diff
@@ -1,9 +1,5 @@
-const SimpleGit = require("simple-git");
-const simpleGits = [];
-const fs = require("fs");
-const path = require("path");
+const git_Helper = require(__dirname + "/git_helper.js");
```

Comment:
I'd rename that variable to GitHelper to stay consistent with the other helpers (the filename is ok)

Reply:
done.